Assets/MixedRealityToolkit.Extensions/Socketer/Scripts/Socketer.cs

```diff
@@ -0,0 +1,173 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if (UNITY_WSA || WINDOWS_UWP) && !UNITY_EDITOR
+#define NETFX_CORE
+#endif
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Sockets;
+
+#if !NETFX_CORE
+using System.Threading;
+#else
+using System.Linq;
+using System.Threading.Tasks;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Networking;
+using Windows.Storage.Streams;
+using Windows.Networking.Connectivity;
+using System.Threading;
+#endif
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.Socketer
+{
+#if UNITY_EDITOR || UNITY_STANDALONE || UNITY_WSA || UNITY_IOS
```

Comment:
Are these the only types supported by the Socket plugin? This seems like the list of platforms currently supported by the Spectator view feature, though this is somewhat independent of that. 

Also, with this number of platforms, I feel a custom define is necessary.

Reply:
This needs to be removed or this won't compile for android. I'll update this now.